traffic_portal/app/src/common/modules/form/server/form.server.tpl.html

```diff
@@ -34,8 +34,8 @@
                     <li><a ng-click="confirmStatusUpdate()">Update Status</a></li>
                     <li ng-show="isCache(server) && !server.updPending"><a ng-click="queueServerUpdates(server)">Queue Server Updates</a></li>
                     <li ng-show="isCache(server) && server.updPending"><a ng-click="clearServerUpdates(server)">Clear Server Updates</a></li>
-                    <li ng-show="isEdge(server) || isCache(server)" class="divider"></li>
-                    <li name="viewCapabilitiesMenuItem"><a ng-click="viewCapabilities()">Manage Capabilities</a></li>
+                    <li ng-show="isCache(server)" class="divider"></li>
+                    <li ng-show="isCache(server)" name="viewCapabilitiesMenuItem"><a ng-click="viewCapabilities()">Manage Capabilities</a></li>
                     <li><a ng-click="viewDeliveryServices()">Manage Delivery Services</a></li>
```

Comment:
I guess the `viewDeliveryServices` menu item should have had the `isEdge(server)` check because only edges can be assigned to delivery services.

Reply:
it's all very confusing :p